freqtrade/exchange/exchange.py

```diff
@@ -69,6 +69,7 @@ class Exchange:
         "trades_pagination_arg": "since",
         "l2_limit_range": None,
         "l2_limit_range_required": True,  # Allow Empty L2 limit (kucoin)
+        "ccxt_futures_name": "swap"
```

Comment:
is the default for this in ccxt swap, not futures? that's kindof surprising me ... (but not a problem - just a question).

Reply:
> is the default for this in ccxt swap, not futures? that's kindof surprising me (but not a problem - just a question) as i was of the opinion that "futures" is the default, and some exchanges use swap ... 😆

So futures is actually the incorrect term for perpetual futures, perpetual futures are actually perpetual swaps, and the only ones that are actually futures are the ones with an expiry date, I had a big conversation about it with the other CCXT people. They also like going with single words so that's why it's swap, there's a chance that the name could change, but for now it's swap. According to the other CCXT people, Binance is making a slow switch away from calling them futures, the ccxt binance futures code was the first written though, so they went with `future` and `delivery`, but we're trying to think of a way of switching over to the unified naming